### ISSUE TITLE
Can pre-trained scripts be provided

### ISSUE THREAD

**Issue by SELECT-FROM** [TranslationalBioinformaticsUnit/scbert-reusability#1]
Can you provide pre-train script? Thanks

**Comment by GlancerZ** [comment]
> Can you provide pre-train script? Thanks

Do u have pre-train code right now?

**Comment by SELECT-FROM** [comment]
> > Can you provide pre-train script? Thanks
> 
> Do u have pre-train code right now?

Yes, I already have pre train codes

**Comment by GlancerZ** [comment]
> > > Can you provide pre-train script? Thanks
> > 
> > 
> > Do u have pre-train code right now?
> 
> Yes, I already have pre train codes
could u send it to me?

**Comment by SELECT-FROM** [comment]
> could u send it to me?

There are still some issues with the scBERT pre training script that I have implemented myself, and the biggest problem is that the loss does not converge. Since I am no longer using scBERT, I did not further explore the writing of pre training scripts.
